src/main/kotlin/org/rucca/cheese/model/GetMyTeams200ResponseDataDTO.kt

```
@@ -0,0 +1,13 @@
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import io.swagger.v3.oas.annotations.media.Schema
+import javax.validation.Valid
+
+/** @param teams */
+data class GetMyTeams200ResponseDataDTO(
+    @field:Valid
+    @Schema(example = "null", required = true, description = "")
+    @get:JsonProperty("teams", required = true)
+    val teams: kotlin.collections.List<TeamDTO>
+) {}
```

detekt

[warning] The class or object GetMyTeams200ResponseDataDTO is empty.

(detekt.empty-blocks.EmptyClassBlock)